sharedutils/src/main/java/org/zaproxy/zap/sharedutils/binlist/BinRecord.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.sharedutils.binlist;
+
+public class BinRecord {
+    String bin;
+    String brand;
+    String category;
+    String issuer;
+
+    BinRecord(String bin, String brand, String category, String issuer) {
+        this.bin = bin;
+        this.brand = brand;
+        this.category = category;
+        this.issuer = issuer;
+    }
+
+    public String getBin() {
+        return bin;
+    }
+
+    public String getBrand() {
+        return brand;
+    }
+
+    public String getCategory() {
+        return category;
+    }
+
+    public String getIssuer() {
+        return issuer;
+    }
+
+    public String toString() {
```

Comment:
I guess this method isn't really needed. I had created it while building/testing. But then ended up doing something similar in the individual scan packages so that the field names would be local and translated (once CrowdIn is back in action).